tests/orm/nodes/test_node.py

```diff
@@ -837,6 +837,10 @@ def test_tab_completable_properties(self):
 class TestNodeDelete:
     """Tests for deleting nodes."""
 
+    # TODO: Why is this failing for SQLite??
+    # sqlalchemy.orm.exc.ObjectDeletedError: Instance '<DbNode at 0x7f6a68845990>' has been deleted,
+    # or its row is otherwise not present.
+    @pytest.mark.requires_pgsql
```

Comment:
My first guess would be that it might have something to do with the `backend.transaction()` call on 863. Not sure if this is directly inherited from the psql_dos implementation but transactions may work different in sqlite. I would start looking there first

Reply:
I won't have time for digging into this probably. I'll create a separate issue.

Reply:
That is fine, but we will want to prioritize this because to me it suggests there might be a bug lurking with node deletion for sqlite profiles. It is very likely that this hasn't been tested properly yet.

Reply:
Opened https://github.com/aiidateam/aiida-core/issues/6436